x/accounts/keeper.go

```diff
@@ -71,24 +70,22 @@ type InterfaceRegistry interface {
 
 func NewKeeper(
 	cdc codec.BinaryCodec,
-	ss store.KVStoreService,
+	env appmodule.Environment,
 	es event.Service,
-	hs header.Service,
-	bs branch.Service,
-	gs gas.Service,
 	addressCodec address.Codec,
 	signerProvider SignerProvider,
 	execRouter MsgRouter,
 	queryRouter QueryRouter,
 	ir InterfaceRegistry,
 	accounts ...accountstd.AccountCreatorFunc,
 ) (Keeper, error) {
-	sb := collections.NewSchemaBuilder(ss)
+	storeService := env.KVStoreService
+	sb := collections.NewSchemaBuilder(storeService)
 	keeper := Keeper{
-		storeService:     ss,
+		storeService:     storeService,
```

Comment:
to be consistent i think we should store env and use that where its needed, it causes some more changes but keeps it simpler for what is needed to be store

Reply:
https://github.com/cosmos/cosmos-sdk/pull/19398/files#diff-eff63269a2122bd0bc1c08f3d029aa99812aa47ce9fd85ef531dc3e04327ffc5L52 we just did the change here and agree and think this cleaner

Reply:
Yeah this makes sense to me. Thanks for the link Sam.